### ISSUE TITLE
feature: EIP4844, enable circuit and contract support for Blob's kzg

### ISSUE THREAD

**Issue by anylots** [scroll-tech/zkevm-circuits#923]
### Describe the feature you would like
This is an amazing project in the web3 field, and you have done a great job.
And here are some questions and discussions:
How is the progress of scroll's support for circuits eip4844？
When 4844 is launched, how do we verify that the kzg commitment from the data blob and the kzg commitment from the proof are based on the same txs？

This blog post [https://scroll.io/blog/kzg] discusses the previous plan, but the implementation details of 4844 have been fully determined so far. To achieve this feature, it is necessary to modify the verification contract and zkevm circuit. 
Do the scroll team have specific research details on this feature？

**Comment by lispc** [comment]
yes we are designing & implementing circuit support for eip4844. Very likely it will be finished next several months

**Comment by 0xShankar** [comment]
I see that the team is working on circuit support for EIP-4844, which is crucial for future scalability. One thing I’d love to contribute towards is improving the verification process between KZG commitments from data blobs and the proof commitments. Has there been any exploration into optimizing the storage of these commitments to streamline the verification in low-latency environments? I’d be happy to assist in designing a more efficient verification method or testing potential bottlenecks using sample blobs in a simulated environment. Also, would there be any focus on modularizing circuit components for ease of future upgrades?
